app/util/AWS.scala

```diff
@@ -34,6 +34,8 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val kinesisStreamName = config.getString("aws.kinesis.streamName").get
 
+  lazy val kinesisReindexStreamName = config.getString("aws.kinesis.reindexStreamName").get
```

Comment:
Thanks @LATaylor-guardian , no I hadn't realised that.

With that in mind though, I think it still makes sense to have the option in the code to use different streams, but we can just put the same stream in the configuration file. Does that make sense?